integrationExamples/gpt/hello_world.html

```diff
@@ -9,8 +9,10 @@
 
 <html>
   <head>
+      <script async src="../../build/dev/prebid.js"></script>
+      <script async src="https://www.googletagservices.com/tag/js/gpt.js"></script>
       <script>
-        var PREBID_TIMEOUT = 3300;
+        var FAILSAFE_TIMEOUT = 3300;
```

Comment:
+1 on renaming this, had heard people confused thinking it was bidder timeout and others!